packages/block-editor/src/components/url-input/index.js

```diff
@@ -212,6 +218,25 @@ class URLInput extends Component {
 		}
 	}
 
+	onFocus( event ) {
+		const { suggestions } = this.state;
```

Comment:
Is there a reason `this.state` gets destructured but not `this.props`?

Reply:
+1. I wouldn't really notice (and don't mind it) but `event` and `props` are both also used go get a single value from the object and are inlined

Reply:
I had copy/pasted from `onChange` and then added in the `const { suggestions } = this.state;` because I needed access to the suggestions array. Then I didn't go back and refactor. That's the _why_ :) 

Do you think I should:
1. Refactor to have both `this.state` and `this.props` to be destructured at the top of the method
2. Refactor to remove `this.state` being destructured and use `this.state.suggestions` inline in the if statement below
3. Merge as is

Reply:
My vote is for 1. Not sure if there are coding guidelines for things like this yet.

Reply:
That feels like the cleanest option to me as well.